yarn.lock

```diff
@@ -7045,6 +7045,13 @@ debug@^3.1.0, debug@^3.1.1, debug@^3.2.7:
   dependencies:
     ms "^2.1.1"
 
+debug@^4.3.2, debug@^4.3.4:
```

Comment:
nit: how come this ended up duplicated away from `debug@4, debug@^4.0.1, debug@^4.1.0, debug@^4.1.1, debug@^4.2.0, debug@^4.3.1`

Reply:
dunno. I did dedup to clean up this and globby